minimesos/src/main/java/com/containersol/minimesos/mesos/MesosAgent.java

```diff
@@ -56,16 +58,17 @@ public CreateContainerCmd getBaseCommand() {
         String hostDir = MesosCluster.getHostDir().getAbsolutePath();
 
         return dockerClient.createContainerCmd( getMesosImageName() + ":" + getMesosImageTag() )
-                .withName( getName() )
-                .withPrivileged(true)
-                .withEnv(createMesosLocalEnvironment())
-                .withPid("host")
-                .withLinks(new Link(getZooKeeperContainer().getContainerId(), "minimesos-zookeeper"))
-                .withBinds(
-                        Bind.parse("/var/run/docker.sock:/var/run/docker.sock"),
-                        Bind.parse("/sys/fs/cgroup:/sys/fs/cgroup"),
-                        Bind.parse(hostDir + ":" + hostDir)
-                );
+	    .withName( getName() )
+	    .withPrivileged(true)
+	    .withEnv(createMesosLocalEnvironment())
+	    .withPid("host")
+	    .withLinks(new Link(getZooKeeperContainer().getContainerId(), "minimesos-zookeeper"))
+	    .withBinds(
+		       Bind.parse("/var/run/docker.sock:/var/run/docker.sock"),
+		       Bind.parse("/sys/fs/cgroup:/sys/fs/cgroup"),
+		       Bind.parse(String.format("%s:%s:rw", hostDir + "/.minimesos/sandbox-" + getClusterId() + "/" + MesosCluster.SLAVE_MAPPING_DIR + this.getUuid(), MESOS_AGENT_SANDBOX_DIR)),
+		       Bind.parse(hostDir + ":" + hostDir)
+		       );
```

Comment:
minimesos.host.dir - is the ${PWD}/.minimesos 
user.dir - ${HOME}/.minimesos

If that's correct, then we should create the sandbox under minimesos.host.dir, which is the case now.
Let me know if that's the case.